client/src/components/OverviewFolder/Overview.jsx

```diff
@@ -35,36 +36,18 @@ const Overview = (props) => {
   return (
     <div>
       <h2>Overview</h2>
-      <div id="overview">
-        <div id="imageGallery">
-          <img id="main" src={mainPhoto} onClick={() => {console.log(mainPhoto)}}></img>
-          <CarouselThumbnail photos={itemStyle.photos}/>
-        </div>
-        <div id="productInformation">
-       <div> &#9733;&#9733;&#9733;&#9734;&#9734; Read all [#] reviews</div>
-          <div>{item.category}</div>
-          <div>{item.name}</div>
-          <div>{itemStyle.original_price}</div>
-          <div>Style > {itemStyle.name}</div>
-            <Style photos={itemStyle.photos}/>
-            <div id="sizeDropdown">
-              <Dropdown placeHolder="Select Size" />
-              <Dropdown placeHolder="Quantity" />
+        {item && itemStyle ?
```

Comment:
If you want to get rid of ternary operator... I would create an ImageGallery component that will house the main photo and the carousel component. The formatting and setting main image can get passed to ImageGallery and then passed to Carousel as needed.

then you can just have:
<ImageGallery.... />
<ProductInformation .../>